msupply-foundation/open-msupply
# 2998 inbound returns list view

Fixes #2998

# 👩🏻‍💻 What does this PR do? 
 

Adds the customer returns list view

![Screenshot 2024-02-16 at 4 41 54 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/03f72ba4-7fdb-4ca5-bf05-c844f7d4f81e)



# 🧪 How has/should this change been tested? 


- Go to Distribution > Inbound Returns
- See table view
- Selecting lines + delete calls the mock delete mutation
- Export to CSV works
- Create return button opens the customer search modal, but doesn't go any further after that yet :)

Note: Same as supplier returns list PR, I think the filters work, but we'll know once the API actually returns data based on what we pass in 😁
